lib/python/gunittest/main.py

```diff
@@ -135,6 +135,11 @@ def main():
     parser.add_argument('--output', dest='output', action='store',
                         default='testreport',
                         help='Output directory')
+    parser.add_argument('--test_list_in',
```

Comment:
here you are adding two mandatory arguments to a CLI utility. 
I am not saying that breaking APIs is necessarily a problem but this should be discussed first.

Reply:
I also prefer not breaking APIs if possible, so I changed it to use default values